src/helpers/calculations.js

```diff
@@ -0,0 +1,24 @@
+export function getLifeCount(date, maxAge = 90) {
```

Comment:
Answering your question, yes this is fine, we can put it later under some setting